src/ethereum/prague/utils/message.py

```diff
@@ -78,6 +79,11 @@ def prepare_message(
     message: `ethereum.prague.vm.Message`
         Items containing contract creation or message call specific data.
     """
+    accessed_addresses = set()
+    accessed_addresses.add(caller)
+    accessed_addresses.update(PRE_COMPILED_CONTRACTS.keys())
+    accessed_addresses.update(preaccessed_addresses)
+
```

Comment:
This change will need to be backported, yes?

Reply:
Yes. Added to #1026